onefuse/api_client.go

```diff
@@ -185,6 +185,7 @@ type StaticPropertySet struct {
 	Name        string                 `json:"name,omitempty"`
 	Description string                 `json:"description,omitempty"`
 	Properties  map[string]interface{} `json:"properties,omitempty"`
+	Raw         string
```

Comment:
dont you need your little `json` thing here?

Reply:
Nope. This is only used within Terraform and it is never serialized or de-serialized so we don't add any Json annotations. If this was sent to OneFuse or parsed from a OneFuse response we would add a `json:"..."` annotation.